mobileapp/components/home/GameList.tsx

```diff
@@ -19,7 +19,9 @@ export const GameList = () => {
       {games && (
         <FlatList
           data={games}
-          renderItem={({ item }) => <GameListItem id={item.id} />}
+          renderItem={({ item }) => (
+            <GameListItem id={item.id} navigation={navigation} />
```

Comment:
I'm surprised that you need to pass down the `navigation` instance. Isn't there a hook to grab it from anywhere? Alternatively, use a React Context to make it available to all the descendants of the main App component.

Reply:
I simply followed the [doc](https://reactnative.dev/docs/navigation#usage). 

@paulSanchezMarmelab suggested me to use a context too, so i will do this

Reply:
no, not urgent